extensions/blocks/tiled-gallery/index.js

```diff
@@ -208,6 +208,10 @@ export const settings = {
 		_x( 'images', 'block search term', 'jetpack' ),
 		_x( 'photos', 'block search term', 'jetpack' ),
 		_x( 'pictures', 'block search term', 'jetpack' ),
+		_x( 'square', 'block search term', 'jetpack' ),
+		_x( 'circle', 'block search term', 'jetpack' ),
+		_x( 'mosaic', 'block search term', 'jetpack' ),
+		_x( 'masonry', 'block search term', 'jetpack' ),
```

Comment:
Should we add `columns` here, as it's another gallery type we offer, or would that be confusing since it would come up with the column blocks? 

I am also not quite sure about adding masonry in there since our Tiled Gallery block does not support the masonry layout yet (it may when #14780 lands).